app/views/users/index.html.erb

```diff
@@ -1,6 +1,17 @@
-<h1>All Users</h1>
-<ul>
+<h1>Users</h1>
+<ul class="users-list">
   <% @users.each do |user| %>
-    <li><%= user.name %></li>
+  <div class="list">
+    <div class="user-image">
+        <img src="path_to_default_user_image.jpg" alt="User Image">
+      </div>
+    <li class="user-container">
+      
```

Comment:
- [ ] Kindly ensure you fix the path to the user Image for your views. I was able to add the image links but they were unable to be displayed. Kindly fix this.

Video demo
[cinnamon-2023-12-02T180232+0200.webm](https://github.com/jlberbesi/blog-app/assets/81932478/051a98b1-b690-4bdc-b688-5549018950a7)

Reply:
![imagen](https://github.com/jlberbesi/blog-app/assets/88105477/4a30be6a-e129-41c3-83b5-313407d8ba3a)
![Imagen de WhatsApp 2023-12-02 a las 13 42 57_d9902d66](https://github.com/jlberbesi/blog-app/assets/88105477/26f2ff9f-f73d-4c8b-8582-1c5ff8d99aac)
![Imagen de WhatsApp 2023-12-02 a las 13 43 10_12e03a6d](https://github.com/jlberbesi/blog-app/assets/88105477/30a9f694-1b4b-4048-ad57-9ac29692a5e7)